srcpkgs/asciinema/template

```diff
@@ -3,12 +3,12 @@ pkgname=asciinema
 version=2.0.2
 revision=3
 build_style=python3-module
-pycompile_module="asciinema"
 hostmakedepends="python3-setuptools"
 depends="$hostmakedepends"
+checkdepends="python3-pip"
 short_desc="Record and share your terminal sessions, the right way"
 maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
-license="GPL-3"
+license="GPL-3.0-or-later"
 # changelog="https://github.com/asciinema/asciinema/blob/master/CHANGELOG.md"
```

Comment:
changelog is a valid field, please use it

Reply:
Yeah it is! I don't know why the original author felt the need to comment it out but I'll pull it back in.

Reply:
At one point we had to comment changelog because the feature was slightly broken I believe.  That was years ago and might as well be fixed now.

Reply:
it is indeed undocumented in Manual.md, yet there are +600 templates using `changelog=`